nextflow_schema.json

```diff
@@ -409,6 +413,12 @@
             "description": "",
             "default": "",
             "properties": {
+                "binning_map_mode": {
+                    "type": "string",
+                    "default": "all",
+                    "description": "Defines mapping strategy to compute co-abundances for binning, i.e. which samples will be mapped against assembly.",
+                    "help_text": "Available: 'all', 'group' or 'own'. Note that 'own' cannot be specififed in combination with --coassemble_group."
```

Comment:
Probably add more explanation, e.g. that bowtie processes increase exponentially with samples or such.

Reply:
done